_data/dcavadia/dcavadia.md

```diff
@@ -0,0 +1,6 @@
+---
+name: Daniel Cavadia
+institution: Universidad Central de Venezuela
+quote: He say he a shooter but I can’t tell by his stance
```

Comment:
We need you to change the quote to fit our CoC

Reply:
Ok, i will

Reply:
It was a reference to a game but i understand the problem

Reply:
"The Eyes, Chico, They Never Lie"